NEWS

```diff
@@ -91,6 +91,12 @@ PHP                                                                        NEWS
 . PDO SQLite:
   . Fixed bug #38334 (Proper data-type support for PDO_SQLITE). (Nikita)
 
+- PgSQL:
+  . Convert resource<pgsql link> to object \PgSql. (Máté)
+  . Convert resource<pgsql result> to object \PgSqlResult. (Máté)
+  . Convert resource<pgsql large object> to object \PgSqlLob. (Máté)
+  . Convert resource<pgsql string> to object \PgsqlString. (Máté)
```

Comment:
Oh, very good idea! This could serve as a good precedent for the RFC. And should `\PgSql` become `PgSql\Connection`?

P.S. 1: I've just noticed that I got the case of `Sql` wrong in `PgsqlString`
P.S. 2: We could make a similar namespacing change retroactively to LDAP. So that `LDAP` could be called as `LDAP\Connection` (almost as your alternative name proposal). This way naming of these new classes could be more consistent.

Reply:
Yes, I think `PgSql\Connection` makes sense. And changing LDAP is also a good idea.